### PR TITLE
Fixes for inducer/sumpy#19

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: pytential/qbx/fmm.py [hunk 1]
```diff
@@ -394,7 +394,7 @@ def drive_fmm(expansion_wrangler, src_weights, timing_data=None,
     :arg geo_data: A :class:`pytential.qbx.geometry.QBXFMMGeometryData` instance.
     :arg expansion_wrangler: An object exhibiting the
         :class:`boxtree.fmm.ExpansionWranglerInterface`.
-    :arg src_weights: Source 'density/weights/charges'.
+    :arg src_weights: A sequence of source 'density/weights/charges'.
```
Comment (inducer) [inline pytential/qbx/fmm.py:397]:
```suggestion
    :arg src_weight_vecs: A sequence of source 'density/weights/charges'.
```
(Globally)

File: pytential/qbx/fmmlib.py [hunk 2]
```diff
@@ -288,6 +288,8 @@ def qbx_local_expansion_zeros(self):
     @log_process(logger)
     @return_timing_data
     def form_global_qbx_locals(self, src_weights):
+        assert len(src_weights) == 1
+        src_weights = src_weights[0]
```
Comment (inducer) [inline pytential/qbx/fmmlib.py:292]:
```suggestion
        src_weights, = src_weight_vecs
```
(Globally)

**Review by inducer — COMMENTED** [review]
Thanks! Mostly fine, just some style things analogous to boxtree.

**Comment by inducer** [comment]
Thanks! This needs its branch names changed back, and then it's also ready to go in.

**Comment by isuruf** [comment]
Thanks for the review

**Pull request merged**